.gitignore

```diff
@@ -6,3 +6,4 @@ erl_crash.dump
 _DS_Store
 elm-stuff
 doc/
+apps/skateboard/config/dev.exs
```

Comment:
i constantly have to edit mine because i don't use the standard postgres username/password.. @sjahandideh is that why you're adding it to the gitignore?